test/frontend.js

```diff
@@ -263,39 +266,29 @@ describe('Frontend', function () {
         .wait(elT('rulesStep'))
         // set rules step skipped for now
-        .click(elT('nextBtn'))
-        .wait(elT('privacyStep'))
-        .click(elT('nextBtn'))
-        // invite members
-        .wait(elT('searchUser'))
-        .insert(elT('searchUser'), username + '4')
-        .click(elT('addButton'))
-        .wait(elT('member'))
-
-      const invited = await n.evaluate(el => document.querySelectorAll(el).length, elT('member'))
-      should(invited).equal(1)
 
-      await n.click(elT('nextBtn')).wait(elT('summaryStep'))
       // summary page sees group as valid
       const valid = await n.exists(`${elT('finishBtn')}:not(:disabled)`)
       should(valid).equal(true)
       // submit group
       await n.click(elT('finishBtn')).wait(elT('dashboard'))
 
-      const created = await n.evaluate(() => ({
-        groupName: document.querySelector('[data-test="groupName"]').innerText,
-        incomeProvided: document.querySelector('[data-test="minIncome"]').innerText,
-        changePercentage: document.querySelector('[data-test="changePercentage"]').innerText,
-        memberApprovalPercentage: document.querySelector('[data-test="approvePercentage"]').innerText,
-        memberRemovalPercentage: document.querySelector('[data-test="removePercentage"]').innerText
-      }))
-      should(created.groupName).equal(testName)
-      // BUG: TODO: this field should not include the currency
-      //      TODO: the currency should be checked separately via data-test="incomeCurrency"
-      should(created.incomeProvided).equal('$' + testIncome)
-      should(created.changePercentage).equal(testSetting + '%')
-      should(created.memberApprovalPercentage).equal(testSetting + '%')
-      should(created.memberRemovalPercentage).equal(testSetting + '%')
+      // const created = await n.evaluate(() => ({
+      //   groupName: document.querySelector('[data-test="groupName"]').innerText,
+      //   sharedValues: document.querySelector('[data-test="sharedValues"]').innerText,
+      //   incomeProvided: document.querySelector('[data-test="minIncome"]').innerText,
+      //   changePercentage: document.querySelector('[data-test="changePercentage"]').innerText,
+      //   memberApprovalPercentage: document.querySelector('[data-test="approvePercentage"]').innerText,
+      //   memberRemovalPercentage: document.querySelector('[data-test="removePercentage"]').innerText
+      // }))
+      // should(created.groupName).equal(testName)
+      // should(created.sharedValues).equal(testValues)
+      // // BUG: TODO: this field should not include the currency
+      // //      TODO: the currency should be checked separately via data-test="incomeCurrency"
+      // should(created.incomeProvided).equal('$' + testIncome)
+      // should(created.changePercentage).equal(testSetting + '%')
+      // should(created.memberApprovalPercentage).equal(testSetting + '%')
+      // should(created.memberRemovalPercentage).equal(testSetting + '%')
```

Comment:
Is there a reason this was removed?

Can it be either re-added, or an issue created to re-add it?